assets/javascripts/kitten/components/graphics/icons-next/icons-next.stories.js

```diff
@@ -163,19 +163,37 @@ export const AllIcons = ({ colorInput, secondaryColorInput }) => {
           <HeartIconNext color={colorInput} />
         </IconContainer>
         <IconContainer>
-          <GiftIconNext
+          <SunIconNext color={colorInput} />
+        </IconContainer>
+        <IconContainer>
+          <EnvelopeIconNext color={colorInput} />
+        </IconContainer>
+        <IconContainer>
+          <StarIconNext color={colorInput} />
+        </IconContainer>
+        <IconContainer>
+          <EditIconNext color={colorInput} />
+        </IconContainer>
+        <IconContainer>
+          <FilterIconNext color={colorInput} />
+        </IconContainer>
+      </Group>
+
+      <Group title="Bicolore icons">
```

Comment:
Si on continue avec des noms en anglais, je propose plutôt “Bicolor icons”, mais si on veut des noms français on peut aller vers « Icônes bicolores »

```suggestion
      <Group title="Bicolor icons">
```